CMSgov/design-system
# Add spans to html examples with form labels

### Summary
Form label html examples dont match the react FormLabel component. This is because the react component adds an extra span. This isnt necessary for most HTML examples, but can be useful for styling (i.e. MCT uses it to style the form label as uppercase). This PR updates all the HTML examples to include the extra span so that the Mgov Child DS examples are consistent

How it looked before: 
<img width="684" alt="Screen Shot 2020-08-19 at 12 10 41 PM" src="https://user-images.githubusercontent.com/5424713/90668055-11db5980-e215-11ea-981f-12d9413ff8f6.png">
How we want it to look
<img width="599" alt="Screen Shot 2020-08-19 at 12 10 44 PM" src="https://user-images.githubusercontent.com/5424713/90668053-10aa2c80-e215-11ea-8cb5-29ed6e3f06c0.png">

### How to test
- Ensure no visual changes in the core
- Ensure the Mgov child ds site has the same FormLabel styling.
